_events/2023-01-25-htcondor-osg.md

```diff
@@ -21,13 +21,14 @@ header_image: "/images/capital.jpeg"
 <p style="font-size: larger; font-weight: bold;">Don't miss these in-person learning opportunities in beautiful Madison, Wisconsin!</p>
 
 
-Save the dates! For the first time, the OSG Consortium, the HTCondor team, and the Center for High Throughput Computing will be hosting a Joint HTCondor Week - OSG All-Hands Meeting from July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/).
+Save the dates for **Throughput Computing 2023!** For the first time, we will be offerring **Throughput Computing 2023**, which combines the much loved events **HTCondor Week** and the **OSG All-Hands Meeting** into a new joint event to be held July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/). **Throughput Computing 2023** is sponsored by the OSG Consortium, the HTCondor team, and the UW-Madison Center for High Throughput Computing. 
```

Comment:
Not a fan of the marketing-speak of “the much loved events”.

Reply:
i changed to long-standing